### ISSUE TITLE
All Configuration & Constant Events for an Inventory Instrument all show same date or TDB

### ISSUE THREAD

**Issue by spetillo** [WHOIGit/ooicgsn-roundabout#206]
@MarioCarloni Can you go on Staging and take a look at CGINS-ADCPTF-20530 in Inventory? I had deployed this instrument on deployment T0003 (formerly named "T0001" in the first screenshot) and then T0002 on Pioneer.

I'm seeing duplication of the deployment date in the Constant History & Configuration History for T0003 (formerly "T0001", shows 4/24/20) and T0002 (should be 10/22/18 as set/shown in the Deployments tab, but shows 4/24/20 in Const & Config Tabs).  See first two screenshots.

![Screenshot from 2020-10-16 15-00-03](https://user-images.githubusercontent.com/11924613/96506948-75d6ca00-1226-11eb-8ba0-c4ee2bb9ae66.png)

![Screenshot from 2020-10-16 15-05-16](https://user-images.githubusercontent.com/11924613/96300038-ebdaf700-0fc2-11eb-90fe-b2d9106a2223.png)

Then, when I recovered T0002, they both now show all TBD dates in the Constant History & Configuration History. See third screenshot.

![Screenshot from 2020-10-16 15-19-40](https://user-images.githubusercontent.com/11924613/96506975-7ff8c880-1226-11eb-8616-2910a785e4f5.png)

**Comment by spetillo** [comment]
This looks great now from the Config History & Constant History tabs. Thanks @MarioCarloni !

![Screenshot from 2020-10-22 14-47-37](https://user-images.githubusercontent.com/11924613/96916451-93e63980-1475-11eb-88f6-4b3aca9e28c5.png)

@sbatchelder Now I need the exports to match. I get 2 different exports for CGINS-ADCPTF-20530 for 2020/04/24, even though I should get one for 2020/04/24 and one for 2018/10/22. The values are correct in the two files, but the date on one of them is incorrect.

![Screenshot from 2020-10-22 14-50-58](https://user-images.githubusercontent.com/11924613/96916871-271f6f00-1476-11eb-8b8b-8d5b829a8892.png)

[CalibrationsWithConfigs_exportlog.txt](https://github.com/WHOIGit/ooicgsn-roundabout/files/5425091/CalibrationsWithConfigs_exportlog.txt)

**Comment by sbatchelder** [comment]
fixed this issue. Export dates now key off of deployment_to_field dates. 

A funny thing I noticed, even though created a hotfix branch from release-1.5.1, I did not see Mario's code changes from #221 
(specifically [these changes](https://github.com/WHOIGit/ooicgsn-roundabout/pull/221/commits/e8bf013a2ece491fc3179111829b53d1f67c0c18)). Which is a little weird, but did not affect my fix.

How exactly do these changes to 1.5.1 get propagated forward to 1.6 and onward??

**Comment by spetillo** [comment]
@sbatchelder Mario's changes got merged into v1.5.1 with #222 instead of #221. I just checked that they are in the v1.5.1 branch. Also, your changes look good, so we're good to release!

Once v1.5.1 is released, everything in `release-1.5.1` should get merged forward into v1.5.2 and v1.6.0+ branches to propagate forward all changes. Similarly, v1.5.2 changes should be regularly merged into v1.6.0+ branches to keep things in step. @ethanandrews should be able to tell you how/when this is happening.
